src/jit/gentree.h

```diff
@@ -940,6 +940,9 @@ struct GenTree
 
 #define GTF_ARRLEN_ARR_IDX 0x80000000 // GT_ARR_LENGTH -- Length which feeds into an array index expression
 
+#define GTF_LIST_AGGREGATE 0x80000000 // GT_LIST -- Indicates that this list should be treated as an
```

Comment:
Then perhaps GTF_LIST_FIELD would be better. I just don't like the idea of mixing "struct" and "aggregate" as the term we use to describe the same thing.

---
In reply to: [76495984](https://github.com/dotnet/coreclr/pull/6932#discussion_r76495984) [](ancestors = 76495984)